workflow/scripts/ScanITD.py

```diff
@@ -0,0 +1,564 @@
+#!/usr/bin/env python3
```

Comment:
I'd rather would want the modifications upstream and us use scanITD via a bioconda package.